lib/CModHandler.h

```diff
@@ -256,6 +274,13 @@ class DLL_LINKAGE CModHandler
 	void loadMods(std::string path, std::string parent, const JsonNode & modSettings, bool enableMods);
 	void loadOneMod(std::string modName, std::string parent, const JsonNode & modSettings, bool enableMods);
 public:
+	
+	class Incompatibility: public std::logic_error

```

Comment:
Do we catch this exception? If not - we need to log error before throwing it. Otherwise users might only observe a crash without any clues what caused it.

Reply:
This is caught on server side. Unfortunatelly I don't know how to bypass it to client . The intention is to catch specific type of excepton and show message to user. If you know how to do it please advice